dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESMappingAPIImpl.java

```diff
@@ -332,7 +319,98 @@ public String toJson(Contentlet con) throws DotMappingException {
 		}
 	}
 
-    /**
+	private void generateBinaryMetadata(final Contentlet contentlet,
+										final StringWriter stringWriter,
+										final Map<String, Object> mapLowered,
+										final Field field) throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4960665)